sympy/logic/inference.py

```diff
@@ -132,7 +132,7 @@ def valid(expr):
     return not satisfiable(Not(expr))
 
 
-def pl_true(expr, model={}, deep=False):
+def pl_true(expr, model=None, deep=False):
```

Comment:
Safer to not have empty dicts or lists as default arguments, although it may not always be a problem.